server/server.go

```diff
@@ -525,6 +525,12 @@ func (server *Server) prepareServer(entryPointName string, router *middlewares.H
 		return nil, err
 	}
 
+	middlewaresStr := ""
+	for _, handler := range negroni.Handlers() {
+		middlewaresStr += reflect.TypeOf(handler).String() + " "
```

Comment:
Yep, you are right, as we don't have middlewares created in `loadConfig`, it is a bit useless. I will remove it.